### PR TITLE
Handle case of not found reset token

### PR DESCRIPTION
**Pull request by lauraluiz**

### CODE REVIEW

File: my-account/app/com/commercetools/sunrise/myaccount/authentication/recoverpassword/reset/SunriseResetPasswordController.java [hunk 1]
```diff
@@ -41,13 +43,13 @@ public SunriseResetPasswordController(final ContentRenderer contentRenderer, fin
...
     @EnableHooks
-    @SunriseRoute(SimpleRecoverPasswordReverseRouter.RESET_PASSWORD_PAGE)
+    @SunriseRoute(RecoverPasswordReverseRouter.RESET_PASSWORD_PAGE)
     public CompletionStage<Result> show(final String languageTag, final String resetToken) {
         return showFormPage(resetToken, formData);
     }
 
     @EnableHooks
-    @SunriseRoute(SimpleRecoverPasswordReverseRouter.RESET_PASSWORD_PROCESS)
+    @SunriseRoute(RecoverPasswordReverseRouter.RESET_PASSWORD_PROCESS)
```
Comment (katmatt) [inline my-account/app/com/commercetools/sunrise/myaccount/authentication/recoverpassword/reset/SunriseResetPasswordController.java]: Good catch 👍

File: my-account/app/com/commercetools/sunrise/myaccount/authentication/recoverpassword/reset/SunriseResetPasswordController.java [hunk 2]
```diff
@@ -41,13 +43,13 @@ public SunriseResetPasswordController(final ContentRenderer contentRenderer, fin
     }
 
     @EnableHooks
-    @SunriseRoute(SimpleRecoverPasswordReverseRouter.RESET_PASSWORD_PAGE)
+    @SunriseRoute(RecoverPasswordReverseRouter.RESET_PASSWORD_PAGE)
```
Comment (katmatt) [inline my-account/app/com/commercetools/sunrise/myaccount/authentication/recoverpassword/reset/SunriseResetPasswordController.java]: Good catch 👍

**Comment by lauraluiz** [comment]
Yes, it's a hard issue to solve because it only happens when you run sequentially IT in command line (via Intellij it doesn't happen). It's not the first time I met this issue but I've managed to avoid it until now :)

**Comment by lauraluiz** [comment]
Solved! I remembered that I just needed to not parallelize the tests :D

**Comment by katmatt** [comment]
I see 👍

**Pull request merged**